### PR TITLE
Last forms

### PR DESCRIPTION
**Pull request by Drakevr**
everything looks rather OK-ish. 
do confirm before blindly merging of course.

### CODE REVIEW

File: models/forms/lesson_entry10.go [hunk 1]
```diff
@@ -14,3 +14,52 @@ type LessonCreateReportFormEntry10 struct {
...
+
+func (f *LessonCreateReportFormEntry10) Update(lessonId int, akademicYearId int) {
+query := `UPDATE LessonCreateReportFormEntry10 SET
+	dia8esimi_ekpedeutikh_ypodomh_tou_ma8imatos_ai8ouses_didaskalias_pou_xrisimopoiountai = $1,
+	dia8esimi_ekpedeutikh_ypodomh_tou_ma8imatos_ergasthria_pou_xrhsimopoiountai_gia_to_sugkekrimeno_ma8ima = $2,
+	dia8esimi_ekpedeutikh_ypodomh_tou_ma8imatos_einai_dia8esima_ta_ergasthria_tou_ma8imatos_gia_xrisi_ektos_programmatismenon_wrwn = $3,
+	dia8esimi_ekpedeutikh_ypodomh_tou_ma8imatos_spoudasthria = $4,
+	dia8esimi_ekpedeutikh_ypodomh_tou_ma8imatos_xrhsimopoihtai_ekpedeutiko_logismiko_kai_poio = $5,
+	dia8esimi_ekpedeutikh_ypodomh_tou_ma8imatos_yparxei_ikanopoihtiki_ypostiriksi_tou_ma8imatos_apo_tin_biblio8iki = $6,
+	dia8esimi_ekpedeutikh_ypodomh_tou_ma8imatos_pws_krinete_sunolika_tin_dia8esimi_ekpedeutiki_ypodomi = $7,
```
Comment (kokeroulis) [inline models/forms/lesson_entry10.go]: L26: comma?

File: models/forms/lesson_entry10.go [hunk 2]
```diff
@@ -14,3 +14,52 @@ type LessonCreateReportFormEntry10 struct {
...
+
+func (f *LessonCreateReportFormEntry10) Load(lessonId int, akademicYearId int) {
+query := `SELECT
+	dia8esimi_ekpedeutikh_ypodomh_tou_ma8imatos_ai8ouses_didaskalias_pou_xrisimopoiountai,
+	dia8esimi_ekpedeutikh_ypodomh_tou_ma8imatos_ergasthria_pou_xrhsimopoiountai_gia_to_sugkekrimeno_ma8ima,
+	dia8esimi_ekpedeutikh_ypodomh_tou_ma8imatos_einai_dia8esima_ta_ergasthria_tou_ma8imatos_gia_xrisi_ektos_programmatismenon_wrwn,
+	dia8esimi_ekpedeutikh_ypodomh_tou_ma8imatos_spoudasthria,
+	dia8esimi_ekpedeutikh_ypodomh_tou_ma8imatos_xrhsimopoihtai_ekpedeutiko_logismiko_kai_poio,
+	dia8esimi_ekpedeutikh_ypodomh_tou_ma8imatos_yparxei_ikanopoihtiki_ypostiriksi_tou_ma8imatos_apo_tin_biblio8iki,
+	dia8esimi_ekpedeutikh_ypodomh_tou_ma8imatos_pws_krinete_sunolika_tin_dia8esimi_ekpedeutiki_ypodomi,
```
Comment (kokeroulis) [inline models/forms/lesson_entry10.go]: L51: comma?

File: models/forms/lesson_entry10.go (hunk 1 above)
Comment (Drakevr) [inline models/forms/lesson_entry10.go]: oops, this was drown in a sea of mails and i failed to see it. Good
catch, updated but my connection is acting up so i can't push right
now, will do so once i get to a place with better connectivity.

File: models/forms/lesson_entry10.go (hunk 2 above)
Comment (Drakevr) [inline models/forms/lesson_entry10.go]: idem ditto

**Pull request merged**